src/App.php

```diff
@@ -163,11 +162,18 @@ protected function configureRoutes( \Slim\Slim $slim ) {
 					echo json_encode( array( 'error' => 'Unauthorized' ) );
 					$slim->stop();
 				}
+			},
+			'trailing-slash' => function () use ( $slim ) {
+				// Remove trailing slashes
+				if ( substr( $_SERVER['REQUEST_URI'], -1 ) === '/' ) {
+					$uri = preg_replace( '/\/+$/', '', $_SERVER['REQUEST_URI'] );
```

Comment:
that's what I had originally, but it will remove only one `/` then redirect, so Chrome will error out `tools.wmflabs.org/plagiabot/////////` with too many redirects